ui/src/providers/docker/context.js

```diff
@@ -10,7 +10,7 @@ export const DockerRegistriesContextProvider = ({ children }) => {
       inputDisplay: registry
     })),
     {
-      value: "",
+      value: "docker.io",
```

Comment:
If using an empty string, we will require the handling logic to change in [SelectDockerImageComboBox](https://github.com/gojek/turing/blob/main/ui/src/components/form/docker_image_combo_box/SelectDockerImageComboBox.js#L6), since the repo name and image are being saved and passed around in the same variable, and an empty string should imply that the default registry should be chosen (in this case, Docker Hub can never be chosen as the registry without entering an image value). This issue is described in more detail in the previous PR: https://github.com/gojek/turing/pull/22